.babelrc

```diff
@@ -1,4 +1,5 @@
 {
+    "sourceMaps": "inline",
```

Comment:
This is a compromise for keeping the browserified output. It uses this same babel config, but only exports a single file.

Reply:
this also seems to suggest browserify is reading the files in `lib` spat out by `build:compile` since those are the files that'll now the have source maps at the bottom of them?

Reply:
Empirically the browserified sourcemaps aren't present without this, so I don't know what's going on. I suspect something weird browserify does that webpack would fix.